rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -1062,6 +1068,130 @@ def test_upgrade_v11_v12(globaldb: GlobalDBHandler, messages_aggregator):
+            ('solana/token:4dmKkXNHdgYsXqBHCuMikNQWwVomZURhYvkkX5c4pQ7y', 'Synthetify Token'),
+            (mango_identifier := 'solana/token:MangoCzJ36AjZyKwVj3VnYU4GTonjfVEnJmvvWaxLac', 'Mango'),  # noqa: E501
+            ('solana/token:MERt85fc5boKw3BW1eYdxonEuJNvXbiMbs6hvheau5K', 'Mercurial'),
+            ('solana/token:ATLASXmbPQxBUYbxPsV97usA3fPQYEqzQBUHgiFCUsXx', 'Star Atlas'),
+        ]
+        assert cursor.execute("SELECT COUNT(*) FROM assets WHERE identifier IN ('HODLSOL','TRISIG')").fetchone()[0] == 0  # noqa: E501
+        # token count reduced by 2 due to removal duplicate HODLSOL and TRISIG duplicate entries
+        assert cursor.execute('SELECT COUNT(*) FROM assets').fetchone()[0] == tokens_before - 2
+        assert cursor.execute('SELECT COUNT(*) FROM common_asset_details').fetchone()[0] == tokens_before - 2  # noqa: E501
+        assert cursor.execute("SELECT COUNT(*) FROM assets WHERE type='Y' AND identifier NOT LIKE 'solana%'").fetchone()[0] == 0  # noqa: E501
+        assert cursor.execute('SELECT main_asset FROM asset_collections WHERE id IN (500, 501, 502) ORDER BY id').fetchall() == [  # noqa: E501
+            (cope_identifier,),
+            (raydium_identifier,),
+            (mango_identifier,),
+        ]
+        assert cursor.execute('SELECT asset FROM multiasset_mappings WHERE collection_id IN (500, 501, 502) ORDER BY collection_id').fetchall() == [  # noqa: E501
+            (cope_identifier,),
+            (raydium_identifier,),
+            (mango_identifier,),
+        ]
+        assert cursor.execute('SELECT * FROM user_owned_assets').fetchall() == [
+            (cope_identifier,),
+            (raydium_identifier,),
+            (mango_identifier,),
+        ]
+        assert cursor.execute('SELECT from_asset, to_asset FROM price_history').fetchall() == [
+            (cope_identifier, 'USD'),
+            (mango_identifier, 'ETH'),
+            ('USD', raydium_identifier),
+        ]
+        assert cursor.execute("SELECT exchange_symbol, local_id FROM location_asset_mappings WHERE exchange_symbol IN ('TRISIG', 'TRUMPSOL')").fetchall() == [  # noqa: E501
+            ('TRISIG', trisig_sol_identifier := 'solana/token:BLDiYcvm3CLcgZ7XUBPgz6idSAkNmWY6MBbm8Xpjpump'),  # noqa: E501
+            ('TRUMPSOL', 'solana/token:6p6xgHyF7AeE6TZkSmFsko444wqoP15icUSqi2jfGiPN'),
+            ('TRISIG', trisig_sol_identifier),
+        ]
+        assert cursor.execute("SELECT symbol, local_id FROM counterparty_asset_mappings WHERE symbol IN ('BONK', 'WIF', 'POPCAT')").fetchall() == [  # noqa: E501
+            ('BONK', 'solana/token:DezXAZ8z7PnrnRJjz3wXBoRgixCa6xjnB7YaB1pPB263'),
+            ('WIF', 'solana/token:EKpQGSJtjMFqKZ9KQanSqYXRcF8fBopzLHYxdM65zcjm'),
+            ('POPCAT', 'solana/token:7GCihgDB8fe6KNjn2MYtkzZcRjQy3t9GHdC8uHYmW2hr'),
+        ]
+        assert cursor.execute('SELECT base_asset, quote_asset FROM binance_pairs').fetchall() == [
```

Comment:
you are not checking for the new index here.

But also ... wonder .... did we not have a DB consistency check which checks all tables? The one that uses the DB minimized schema? I guess this would have caught this? Perhaps it does not check indices? If not shouldn't it?